code/modules/scripting/Interpreter/Evaluation.dm

```diff
@@ -1,203 +1,3 @@
 /proc/isobject(x)

```

Comment:
What still uses this?

Reply:
isobject is used a few times in transformer.dm:
https://github.com/ParadiseSS13/Paradise/blob/cad69ff230c6ae7824622cab000dff7171246175/code/game/machinery/transformer.dm#L137